src/http_upload/mod_http_upload.erl

```diff
@@ -122,6 +123,36 @@ get_urls(Host, Filename, Size, ContentType, Timeout) ->
     mod_http_upload_backend:create_slot(UTCDateTime, Token, Filename,
                                         ContentType, Size, NewOpts).
 
+-spec config_spec() -> mongoose_config_spec:config_section().
+config_spec() ->
+    #section{
+        items = #{<<"iqdisc">> => mongoose_config_spec:iqdisc(),
+                  <<"host">> => #option{type = string,
+                                        validate = domain_template},
+                  <<"backend">> => #option{type = atom,
+                                           validate = {module, mod_http_upload}},
+                  <<"expiration_time">> => #option{type = integer,
+                                                   validate = positive},
+                  <<"token_bytes">> => #option{type = integer,
+                                               validate = positive},
+                  <<"max_file_size">> => #option{type = integer,
+                                                 validate = positive},
+                  <<"s3">> => s3_spec()
```

Comment:
This part is currently mandatory - it has to be provided as `s3` is the only backend and it contains mandatory options for it... I would state both in the code and in the docs that it is mandatory - this is the current state. If we add more backends, we could do the validation differently...